changelog.txt

```diff
@@ -14,6 +14,7 @@
 * Fix   - Fix localization issues
 * Tweak - Improve service listing readability
 * Add   - Support UPS as a carrier (beta)
+* Fix   - Services management CSS table layout
```

Comment:
Same comment as: https://github.com/Automattic/woocommerce-services/pull/2121#pullrequestreview-462861940

Reply:
Thanks, done in c2c5153950cae9c20ff959f6e2cde3988eb93afe